tdms/include/unit_test_utils.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file unit_test_utils.h
+ * @brief Functions common to multiple unit tests
+ */
```

Comment:
I was also wondering about pulling out some test utils into a (header, src) pair. My (very very soft) problem is that I'd prefer testing to live together.

Can we have the header in `tests/unit`? Is our build system smart enough for that?

Reply:
Let me experiment on this branch, after looking at the other comments :+1:

Reply:
Turns out `CMake` is smart enough [if you tell it where to look](https://github.com/UCL/TDMS/pull/187/files#diff-a8a22f01e97802a8ec881f48890a12b6feb81a367999801dbb0feb31ffbf47d5R144-R146).